docs/ldml/tr35-personNames.md

```diff
@@ -41,26 +41,924 @@ The LDML specification is divided into the following parts:
+
+    1. **The foreign name is written in the formatting locale’s script.** In that case, the **foreignSpaceReplacement** is substituted for each space in the patterns from the _locale of the name_. Here are examples for Albert Einstein in Japanese and Chinese:
+        * [アルベルト・アインシュタイン](https://ja.wikipedia.org/wiki/%E3%82%A2%E3%83%AB%E3%83%99%E3%83%AB%E3%83%88%E3%83%BB%E3%82%A2%E3%82%A4%E3%83%B3%E3%82%B7%E3%83%A5%E3%82%BF%E3%82%A4%E3%83%B3) 
+        * [阿尔伯特·爱因斯坦](https://zh.wikipedia.org/wiki/%E9%98%BF%E5%B0%94%E4%BC%AF%E7%89%B9%C2%B7%E7%88%B1%E5%9B%A0%E6%96%AF%E5%9D%A6) 
+    2. **The foreign name is written in a different script.** In that case, the patterns from the **locale of the name** are used as is.
+        * [Albert Einstein](https://de.wikipedia.org/wiki/Albert_Einstein) 
+
+In both cases, the ordering may be changed according to the **Name Order for Locales** settings that each locale provides. If the PersonName object does not supply a locale for a name, then a default locale will be derived based on other information (such as the script of the characters in the name fields).
+
+**Known Issue:** The first version of the data doesn’t handle cases with `foreignSpaceReplacement `where the formatting locale doesn’t need spaces between words, but the name locale has the same ordering as the formatting locale. For example, consider where the formatting locale is Thai, and the name is in English, but transliterated into Thai.
+
+To illustrate how foreign space replacement works, consider the following name data. For illustration, the name locale is given in the maximized form: in practice, `ja` would be used instead of `ja_Jpan_JP`, and so on.: For more information, see Likely Subtags [TBD add link].
+
+| name locale   | given    | surname       |
+| ------------- | -------- | ------------- |
+| `de_Latn_CH`  | Albert   | Einstein      |
+| `de_Jpan_CH`  | アルベルト | アインシュタイン |
+| `ja_Jpan_JP`  | Hayao    | Miyazaki      |
+| `ja_Jpan_JP`  | 駿       | 宮崎           |
+
+Suppose the PersonNames formatting patterns for `ja_JP` and `de_CH` contained the following:
+
+| `ja_JP` formatting patterns |
+| --------------------------- |
+| `<personNames>`<br>
+`	<nameOrderLocales order="givenFirst">und</nameOrderLocales>`<br>
+`	<**nameOrderLocales** order="**surnameFirst**">hu **ja** ko vi yue zh **und_JP**</nameOrderLocales>`<br>
+`	<**foreignSpaceReplacement** xml:space="preserve">**・**</foreignSpaceReplacement>`<br>
+`	. . .`<br>
+`	<personName order="**givenFirst**" length="medium" usage="referring" formality="formal">`<br>
+`		<namePattern>{given} {given2} {surname}{suffix}</namePattern>`<br>
+`	</personName>`<br>
+`	. . .`<br>
+`	<personName order="**surnameFirst**" length="medium" usage="referring" formality="formal">`<br>
+`		<namePattern>{surname}{given2}{given}{suffix}</namePattern>`<br>
+`	</personName>`<br>
+`	. . .`<br>
+`</personNames>` | 
+
```

Comment:
Do we really want this as a table?  It looks kind of weird (at least in GitHub's Markdown viewer).  You might just want a header in boldface followed by the XML text inside a ```.  (Same thing with the other similar tables.)

Reply:
It could be pulled out of a table. I think the biggest problem is that inside of ' ticks, the ** don't work.
`order="**givenFirst**"`
==>
`order="`**`givenFirst`**`"`

Reply:
But that's also kind ugly.